protob/messages-tron.proto

```diff
@@ -0,0 +1,207 @@
+syntax = "proto2";
+
+// Sugar for easier handling in Java
+option java_package = "com.satoshilabs.trezor.lib.protobuf";
+option java_outer_classname = "TrezorMessageTron";
+
+/**
+ * Request: Ask device for Tron public key corresponding to address_n path
+ * @next TronPublicKey
+*/
+message TronGetPublicKey {
+    repeated uint32 address_n = 1;  // BIP-32 path to derive the key from master node
+    optional bool show_display = 2; // Optionally show on display before sending the result
+}
+
+/**
+ * Response: Contains Tron public key derived from device private seed
+ * @prev TronGetPublicKey
+*/
+message TronPublicKey {
+    optional bytes public_key = 1;  // Tron public key
+}
+
+/**
+ * Request: Ask device for Tron address corresponding to address_n path
+ * @next PassphraseRequest
+ * @next TronAddress
+ * @next Failure
+ */
+message TronGetAddress {
+    repeated uint32 address_n = 1;  // BIP-32 path to derive the key from master node
+    optional bool show_display = 2; // Optionally show on display before sending the result
+}
+
+/**
+ * Response: Contains Tron address derived from device private seed
+ * @prev TronGetAddress
+ */
+message TronAddress {
+    optional string address = 1;    // Tron address
+}
+
+/**
+ * Request: Ask device to sign Tron transaction
+ * @next TronSignedTx
+ */
+message TronSignTx {
+    repeated uint32 address_n = 1;              // BIP-32 path to derive the key from master node
+    optional TronTransaction transaction = 2;   // Transaction data
```

Comment:
is there any reason this is a separate message, as opposed to having the fields inside `TronSignTx` directly?

Reply:
I have tried to keep it as close as we have in our protocol structure to facilitate the maintenance. Have it updated now.